WooCommerce/WooCommerceTests/ViewRelated/Orders/Order Creation/ProductSelectorViewModelTests.swift

```diff
@@ -135,6 +147,7 @@ final class ProductSelectorViewModelTests: XCTestCase {
         viewModel.sync(pageNumber: 1, pageSize: 25, onCompletion: { _ in })
 
         // Then
+        XCTAssertTrue(syncStatusSpy.contains(.firstPageSync))
```

Comment:
nit: how about asserting on the full array of `syncStatusSpy` so that we can ensure the values stay the same?

Reply:
Definitely, changed on https://github.com/woocommerce/woocommerce-ios/pull/9387/commits/0ed4a6ff0c4475a18f6e6075f88cc4ea3ad0bea0 so we intercept both statuses.